api/_core/bulk.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
+ * This file was generated from OpenSearch API Spec. Do not edit it
+ * manually. If you want to make changes, either update the spec or
+ * the API generator.
```

Comment:
Add something like "by running npm ..." or a mention of a doc like "see dev guide"